Packs/CiscoWSA/Integrations/CiscoWSAV2/CiscoWSAV2.py

```diff
@@ -0,0 +1,1663 @@
+
+        Returns:
+            dict[str, Any]: API response from Cisco WSA.
+        """
+        data = [
+            {"IP_addresses": ip_addresses, "domain_name": domain_name, "order": order}
+        ]
+
+        return self._http_request(
+            "POST", f"{V2_PREFIX}/configure/web_security/domain_map", json_data=data, ok_codes=[HTTPStatus.OK]
+        )
+
+    def domain_map_update(
+        self,
+        domain_name: str,
+        new_domain_name: str | None,
+        ip_addresses: str | None,
+        order: int | None,
+    ) -> dict[str, Any]:
+        """
+        Update domain map.
+
+        Args:
+            domain_name (str): Domain name to update.
+            new_domain_name (str | None): New domain name.
+            ip_addresses (str | None): IP addresses to map.
+            order (int | None): Index of domain map.
+
+        Returns:
+            dict[str, Any]: API response from Cisco WSA.
+        """
+        data = remove_empty_elements(
+            [
+                {
+                    "domain_name": domain_name,
+                    "new_domain_name": new_domain_name,
+                    "IP_addresses": ip_addresses,
+                    "order": order,
+                }
+            ]
+        )
+
+        return self._http_request(
+            "PUT", f"{V2_PREFIX}/configure/web_security/domain_map", json_data=data, ok_codes=[HTTPStatus.OK]
+        )
+
+    def domain_map_delete(self, domain_name: str) -> dict[str, Any]:
+        """
+        Delete domain map.
+
+        Args:
+            domain_name (str): Domain name to delete.
+
+        Returns:
+            dict[str, Any]: API response from Cisco WSA.
+        """
+        data = {"domain_name": domain_name}
+
+        return self._http_request(
+            "DELETE",
+            f"{V2_PREFIX}/configure/web_security/domain_map",
+            json_data=data, ok_codes=[HTTPStatus.OK, HTTPStatus.PARTIAL_CONTENT]
+        )
+
+    def identification_profiles_list(
+        self,
+        profile_names: List[str] | None,
+    ) -> dict[str, Any]:
+        """
+        Get identification profiles.
+
+        Args:
+            profile_names (List[str] | None): Profile names to list.
+
+        Returns:
+            dict[str, Any]: API response from Cisco WSA.
+        """
+        params = assign_params(
+            profile_names=",".join(profile_names) if profile_names else None
+        )
+
+        return self._http_request(
+            "GET", f"{V3_PREFIX}/web_security/identification_profiles", params=params
+        )
+
+    def identification_profiles_create(
+        self,
+        profile_name: str,
+        status: str,
+        description: str,
+        order: int | None,
+        protocols: List[str],
+        proxy_ports: List[str] | None,
+        members_by_subnet: List[str] | None,
+        predefined_url_categories: List[str] | None,
+        custom_url_categories: List[str] | None,
+    ) -> Response:
```

Comment:
Why the return type here is 'Response' and in other request functions it is 'dict[str, Any]'?
We need to be consistent about that. 🕵🏼‍♀️